alphabase/constants/const_files/psm_reader.yaml

```diff
@@ -52,10 +52,10 @@ maxquant:
       - 'K(Dimethyl)'
     'Dimethyl@R':
       - 'R(Dimethyl)'
-    'Dimethyl@Any N-term':
+    'Dimethyl@Any_N-term':
       - '(Dimethyl)'
-    'Acetyl@Protein N-term':
-      - '_(Acetyl (Protein N-term))'
+    'Acetyl@Protein_N-term':
+      - '_(Acetyl (Protein_N-term))'
```

Comment:
The actual unimod names contain a space, so we cannot modify the MQ source PTM names

Reply:
release in 1.3.0, fixed here for 1.4.2: https://github.com/MannLabs/alphabase/pull/241